flink-python/src/main/java/org/apache/flink/python/PythonOptions.java

```diff
@@ -148,4 +149,17 @@
 			"The priority is as following: 1. the configuration 'python.client.executable' defined in " +
 			"the source code; 2. the environment variable PYFLINK_EXECUTABLE; 3. the configuration " +
 			"'python.client.executable' defined in flink-conf.yaml");
+
+	/**
+	 * Whether the memory used by the Python framework is managed memory.
+	 */
+	public static final ConfigOption<Boolean> USE_MANAGED_MEMORY = ConfigOptions
```

Comment:
How about making it an internal config (exclude from documents) since using managed memory does not work well in all cases? I feel that normal users will not need to change it. Correct me if I'm wrong here.

Reply:
I'm not quite sure about this. I think it still has its value to expose it to users, e.g. Python worker uses managed memory by default In 1.10 and it allows Python UDF users of 1.10 to migrate their job to 1.11 by setting this config option as `true` to have the same behavior as 1.10. I guess this is also the similar reason why `state.backend.rocksdb.memory.managed` exists.

Reply:
I see. Makes sense.